Fall-2015/HW-5/p3/p3.md

```diff
@@ -0,0 +1,45 @@
+مساله برج هانوی و روش حل بازگشتي آن را از [ويكی پديا](https://fa.wikipedia.org/wiki/%D8%A8%D8%B1%D8%AC_%D9%87%D8%A7%D9%86%D9%88%DB%8C) مطالعه كنيد. هدف از اين تمرين اين است كه مراحل حل اين مساله برای n حلقه (كه تعداد آن از كاربر گرفته ميشود) را به صورت شكل‌های زير نمايش دهيد.  
+به طور **کلی** (توجه کنید که این کد صرفا برای توضیح منطق الگوریتم آورده شده است) کد شما دارای ساختار زیر خواهد بود:  
+
+    void hanoi ( int nDisk, char start, char temp, char finish )
```

Comment:
Using backtick is better than space for code blocks and also you can specify the language.